chainer/optimizer.py

```diff
@@ -581,7 +589,7 @@ def update(self, lossfun=None, *args, **kwds):
                 self.target.cleargrads()
             else:
                 self.target.zerograds()
-            loss.backward()
+            loss.backward(loss_scale=self.loss_scale)
```

Comment:
I guess this line fails if `set_loss_scale` has not been called. `loss_scale` should have the default value (`None`) and be checked.

Reply:
Thank you for pointing it out!  will fix to initialize that `loss_scale`.